admin/routes/check-form.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const express = require('express')
+const router = express.Router()
+const isAuthenticated = require('../authentication/middleware')
+const rolesConfig = require('../roles-config')
+const checkFormV2Controller = require('../controllers/check-form-v2')
+const featureToggles = require('feature-toggles')
+
+if (featureToggles.isFeatureEnabled('newCheckForm')) {
+  /* Check Form v2 routing */
+  router.get('/upload-and-view-forms', isAuthenticated(rolesConfig.ROLE_TEST_DEVELOPER), (req, res, next) => checkFormV2Controller.uploadAndViewFormsPage(req, res, next))
+  router.get('/upload-new-form', isAuthenticated(rolesConfig.ROLE_TEST_DEVELOPER), (req, res, next) => checkFormV2Controller.uploadCheckFormPage(req, res, next))
```

Comment:
This controller method would make more sense to me if it was called getUploadPage().  In this context "upload" seems to be a verb which sounds like it ought to be doing something.

@GuyHarwood  - do we have guidance on controller names?

Reply:
@jon-shipley routes have been changed based on discussions.